packages/block-library/src/navigation-link/index.php

```diff
@@ -376,3 +376,23 @@ function gutenberg_disable_tabs_for_navigation_link_block( $settings ) {
 }
 
 add_filter( 'block_editor_settings_all', 'gutenberg_disable_tabs_for_navigation_link_block' );
+
+function gutenberg_enable_animation_for_navigation_link_inspector( $settings ) {
+	$current_tab_settings = _wp_array_get(
```

Comment:
Does `$current_tab_settings` need renaming?

Reply:
Thanks for catching this! That variable has been renamed.